clientv3/watch.go

```diff
@@ -52,6 +59,22 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsCreateEvent returns true if the event tells that the key is newly created.
+func IsCreateEvent(e *Event) bool {
```

Comment:
i mean we need a unit test. the one you added is an integration test. basically, that one ensures the interaction between the server and client logic is correct. But it does not ensure that these two funcs behaviors exactly as expected since the input is not controlled by the test directly.